### DISCUSSION TITLE
Feature request: practice slow words

### DISCUSSION THREAD

**Discussion by camsloanftc** [monkeytypegame/monkeytype#1274] [Ideas]
Similar to "practice missed words," this could help you to really practice the words that you are slow at typing, but don't necessarily mis-type. 

I find my left hand is often much slower than my right, and so I would benefit a lot from being able to improve the muscle memory for these slow words, while skipping the ones I can do with ease.

Would love your thoughts on if this is useful, or if there is already some way to do this.

Edit: perhaps there could be a threshold setting for words that fall under a certain speed, or something to that effect, since "slow" is a relative term. Or maybe it's the slowest X% of words typed in that round.

**Comment by unrealapex** [comment]
It would be helpful, but you would be just able to do that just by adding words you are slow at typing to custom tests.

**Reply by camsloanftc** [reply]
True, but I don't really know what words I am slow at typing because I am in the middle of typing them. After the test is completed, I don't think there is a way to view which words I was slow at.

**Reply by unrealapex** [reply]
>I don't think there is a way to view which words I was slow at.

PR #1125 will fix that once @lukew3 has some free time.
